client/app/smallgroup/smallgroup.html

```diff
@@ -139,15 +139,17 @@ <h2 class="group-name">
           <tr>
             <td sort-arrow sortby='datetime' text='Date' sortorder='sortorder'></td>
             <td sort-arrow sortby='bonusDay' text='Type' sortorder='sortorder'></td>
+            <td text ='Attendees'>Attendees</td> 
             <td text='Delete'>Delete</td>
           </tr>
         </thead>
         <tbody>
           <tr ng-repeat="day in smallgroup.dayCodes | orderBy:sortorder">
```

Comment:
This should not be present in fullscreen view. 
![screen shot 2016-10-04 at 3 02 36 pm](https://cloud.githubusercontent.com/assets/5566205/19088354/c418d3e4-8a43-11e6-8b0f-81eed4ef4418.png)